### DISCUSSION TITLE
Github charts are consistently unreliable

### DISCUSSION THREAD

**Discussion by danm** [community/community#63851] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

We're trying to adopt GitHub Projects in our organisation, I'm a huge evangelist for the workflow, but the Project Insights time series charts keep breaking. My delivery lead relies on these charts to complete his work, and when they are unavailable, the discussion of moving back to Jira keeps coming back.

What is the long term plan for Project Insight charts? Are they going to continue to be supported or phased out? Due to how it keeps and remains broken, it doesn't feel like it's a feature taken seriously. I understand the GitHub team use these tools, too, do you spend much time analysing burn-down charts for your development process? 

Curious how the rest of the community measures productivity, as in a number of tickets created vs closed, over time how many tickets are open at one time, time to complete etc etc

thanks all

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Jaytxo** [comment]
We have the same concern. We decided to move to Github projects from Pivotal Tracker to improve our product development workflow. And the main reason to upgrade was that. Now that we started to use it, we realiced that we loosed all the metrics, burn down charts and that stuff.

**Comment by slinna** [comment]
Hi @danm 

_Curious how the rest of the community measures productivity, as in a number of tickets created vs closed, over time how many tickets are open at one time, time to complete etc etc_

You can pull that information from the GitHub API and roll out your own dashboard. Alternatively, you can pick one of the add-ons in the marketplace, such as [Analytics & Reports](https://github.com/marketplace/analytics-reports) by Screenful (disclosure: I'm one of the developers). We have recently added support for GitHub Projects which you can learn more from [this post](https://screenful.com/product-news/2023-06). There's a 30-day free trial so you can give it a try with your own data.

**Reply by danm** [reply]
If we have to roll our own dashboards my EM will murder me! I'll check out some of the third party stuff. 

I need to avoid them saying "Jira did that" in order to get "buy in" from senior leaders

Thanks for the suggestions 🙏🙏

**Comment by GreenPlanet0204** [comment]
We have the same concern. We decided to move to Github projects from Pivotal Tracker to improve our product development workflow

**Comment by Binary-Ninja-007** [comment]
I understand your frustration with the reliability of GitHub's Project Insights time series charts. While I don't have access to GitHub's long-term plans or insights into specific feature development, I can provide some general information.

GitHub continuously updates and improves its features based on user feedback and needs. They prioritize addressing reported issues and enhancing functionality. However, it's worth noting that the availability and reliability of specific features can vary over time.

To get the most accurate and up-to-date information regarding the long-term plan for Project Insight charts, it's best to reach out directly to GitHub's support or consult their documentation and official announcements. They can provide you with specific updates and roadmap information for the feature.

Regarding the GitHub team's usage and analysis of burn-down charts or productivity measurement, as an AI assistant, I don't have insights into their internal processes. However, GitHub provides various tools and integrations that allow users to measure productivity and track progress, such as milestones, labels, issues, and pull requests. These tools can help in monitoring the number of tickets created/closed, time to complete, and overall project progress.

Remember that productivity measurement can vary among teams and organizations. Some adopt custom metrics based on their specific workflows and objectives. It's essential to understand your team's requirements and adjust the metrics accordingly.

If you're considering other productivity measurement options, you can explore other project management tools or integrations that integrate with GitHub or seek community recommendations for tools commonly used in conjunction with GitHub.

Ultimately, I encourage you to provide feedback directly to GitHub regarding the issues you're facing and the importance of reliable Project Insight charts for your organization. They value user feedback, and it helps them prioritize and improve their services.

I hope this information helps, and I wish you the best in finding a suitable solution for measuring productivity in your organization! 😊🚀

**Comment by tohagan** [comment]
You may this useful (not used it)
- https://github.com/thehale/github-projects-burndown-chart

**Comment by andreaslech** [comment]
Still relevant in 2026... the native Insights charts are still unreliable for anything beyond basic counts.

Found a zero-setup alternative recently: GitHub Copilot can generate burndown charts, sprint analysis, and more directly from your Project data. No custom dashboards, no API wrangling.

I put the prompts together in an open-source repo:
[github.com/Flow2CHQ/copilot-agile-metrics](https://github.com/Flow2CHQ/copilot-agile-metrics)

Works in VS Code and in the GitHub web UI. Curious about your thoughts.
